build-system/tasks/dep-check.js

```diff
@@ -318,9 +320,9 @@ function toArrayOrDefault(value, defaultValue) {
 }
 
 /**
- * Flatten array of arrays.
+ * Flatten array of arrays if necessary.
  *
- * @param {!Array<!Array>} arr
+ * @param {Array<Array>|Array} arr
```

Comment:
I think this is a good type because it helps documentation, however, `Array<Array>|Array` is somewhat redundant as it could just be `Array`.